src/components/MesUtilisateurs/InviterUnUtilisateur.tsx

```diff
@@ -0,0 +1,159 @@
+'use client'
+
+import { FormEvent, ReactElement, useContext, useId, useState } from 'react'
+
+import { inviterUnUtilisateurAction } from '../../app/api/actions/inviterUnUtilisateurAction'
+import { clientContext } from '../shared/ClientContext'
+import RadioGroup, { RadioOption } from '../shared/Radio/RadioGroup'
+import TextInput from '../shared/TextInput/TextInput'
+
+const gestionnaires: Array<RadioOption> = [
```

Comment:
```suggestion
const gestionnaires: ReadonlyArray<RadioOption> = [
```

À déplacer en fin de fichier.